pkg/manager/manager_test.go

```diff
@@ -431,3 +435,83 @@ func TestManager_ComputeRollbackDelete(t *testing.T) {
 
 	assert.Equal(t, deletesRoll[0], Test1Cont1ACont2ALeaf2D, "Path should be deleted")
 }
+
+func TestManager_GetTargetState(t *testing.T) {
+	const (
+		device1 = "device1"
+		path1   = "/a/b/c"
+		value1  = "v1"
+		path2   = "/x/y/z"
+		value2  = "v2"
+		badPath = "/no/such/path"
+	)
+	mgrTest, _, _ := setUp()
+
+	//  Create an op state map with test data
+	device1ValueMap := make(map[string]*change.TypedValue)
+	change1 := change.NewTypedValueString(value1)
+	change2 := change.NewTypedValueString(value2)
+	device1ValueMap[path1] = change1
+	device1ValueMap[path2] = change2
+	mgrTest.OperationalStateCache[device1] = device1ValueMap
+
+	// Test fetching a known path from the cache
+	state1 := mgrTest.GetTargetState(device1, path1)
```

Comment:
This supports wildcards too - probably worth testing e.g. GetTargetState with /a/*/c